### PR TITLE
[3.3] Pass boolean pretend instead of array

### LINKED ISSUE #36: [3.3] Migration runDown isn't working
When using reset, or refresh, for example, the package isn't running the runDown migration, instead it's always pretending to run the migration and dumps the result on the console.

I've checked and the content that is being passed to the $pretend variable is
```php
[
    'pretend' => false,
]
```

However on Migrator.php it's checking it like:
```php
if ($pretend) {
    return $this->pretendToRun($instance, 'down');
}
```

I'll try to create a PR for this, but i'm not sure if this also happens with lower versions of the package so I'm posting here beforehand so you can check the issue too...

### PR DESCRIPTION
**Pull request by fernandobandeira**
Fixes #36

### CODE REVIEW

**Comment by fernandobandeira** [comment]
Gonna fix the tests soon :smile:

File: src/Migrator/Factory.php [hunk 1]
```diff
@@ -193,7 +193,7 @@ public function runDown(Model $entity, $database, $pretend = false)
 
         $migrator->setConnection($database);
         $migrator->setEntity($entity);
-        $migrator->rollback($this->getMigrationPath(), ['pretend' => (bool) $pretend]);
+        $migrator->rollback($this->getMigrationPath(), $pretend);
```
Comment (crynobone) [inline src/Migrator/Factory.php]: This code is calling Laravel own `Migrator` which expect an array, not a boolean.

[Migrator::rollback](https://github.com/laravel/framework/blob/5.3/src/Illuminate/Database/Migrations/Migrator.php#L179)

File: src/Migrator/Factory.php [hunk 2]
```diff
@@ -216,7 +216,7 @@ public function runReset(Model $entity, $database, $pretend = false)
 
         $migrator->setConnection($database);
         $migrator->setEntity($entity);
-        $migrator->reset($this->getMigrationPath(), ['pretend' => (bool) $pretend]);
+        $migrator->reset($this->getMigrationPath(), $pretend);
```
Comment (crynobone) [inline src/Migrator/Factory.php]: This should be the only valid changes.

[Migrator::reset](https://github.com/laravel/framework/blob/5.3/src/Illuminate/Database/Migrations/Migrator.php#L228)

Comment (fernandobandeira) [inline src/Migrator/Factory.php]: Ok, will change it

**Comment by fernandobandeira** [comment]
@crynobone I think that's it,also fixed the mock tests 😄

**Pull request merged**